Formula/autorest.rb

```diff
@@ -13,25 +15,24 @@ class Autorest < Formula
     sha256 "da1dc0e3a25b005db13ffbb95b145c060162648ad700998e4814a7969e17cbb1" => :yosemite
   end
 
-  depends_on "mono"
+  depends_on "node"
 
-  resource "swagger" do
-    url "https://raw.githubusercontent.com/Azure/autorest/764d308b3b75ba83cb716708f5cef98e63dde1f7/Samples/petstore/petstore.json"
-    sha256 "8de4043eff83c71d49f80726154ca3935548bd974d915a6a9b6aa86da8b1c87c"
+  resource "petstore" do
+    url "https://raw.githubusercontent.com/Azure/autorest/master/Samples/1b-code-generation-multilang/petstore.yaml"
```

Comment:
This needs to be pinned either to a tagged release or, if there is none, to a commit at least. Otherwise, it will break on the next commit to `master`.

Reply:
@claui et. al: a `brew audit` for this would be 👍